src/controllers/hello-world.api.ts

```diff
@@ -1,30 +1,24 @@
-export const def = {
-    basePath: '/',
-    paths: {
-        '/helloworld': {
-            get: {
-                'x-operation-name': 'helloWorld',
-                parameters: [
-                    {
-                        name: 'name',
-                        in: 'query',
-                        description: 'Your name.',
-                        required: false,
-                        type: 'string',
-                    },
-                ],
-                responses: {
-                    200: {
-                        schema: {
-                            response: {
-                                type: 'string',
-                                description: 'Returns a hello world with your (optional) name.',
-                            },
-                        },
-                    },
-                },
-            },
-        },
-    },
-};
-
+export const apiSpec =
+{
+  "paths": {
+    "/helloworld": {
+      "get": {
+        "x-operation-name": "helloWorld",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "query",
+            "description": "Your name.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a hello world with your (optional) name."
```

Comment:
Thanks, @b-admike  Since hello-world is the intdocuctory tutorial, I'm leaning toward minimizing the contents. But, I may be missing something here.  Will you add more why we want to add schema here, please?